process.md

```diff
@@ -61,126 +62,331 @@ This is for two main reasons:
+project, along with the responsibilities and privileges that come with them.
+Community members generally start at the first levels of the "ladder" and
+advance up it as their involvement in the project grows. Our project members are
+happy to help you advance along the contributor ladder.
+
+Each of the contributor roles below is organized into lists of three types of
+things. "Responsibilities" are things that contributor is expected to do.
+"Requirements" are qualifications a person needs to meet to be in that role, and
+"Privileges" are things contributors on that level are entitled to.
+
+### Community Participant
+
+Description: A Community Participant engages with the project and its community,
+contributing their time, thoughts, etc. Community participants are usually users
+who have stopped being anonymous and started being active in project
+discussions.
+
+- Responsibilities:
+  - Must follow the [Tekton CoC](code-of-conduct.md)
+- How users can get involved with the community:
+  - Participating in community discussions ([GitHub, Slack, mailing list, etc](contact.md))
+  - Helping other users
+  - Submitting bug reports
+  - Commenting on issues
+  - Trying out new releases
+  - Attending community events
+
+### Contributor
+
+Description: A Contributor makes direct contributions to the project and adds
+value to it. Contributions need not be code. People at the Contributor level
+may be new contributors, and they can contribute occasionally.
+
+A Contributor must meet the responsibilities of a Community Participant, plus:
+
+- Responsibilities include:
+  - Follow the project [contributing guide](CONTRIBUTING.md)
+- Requirements (one or several of the below):
+  - Report and sometimes resolve issues
+  - Occasionally submit PRs
+  - Contribute to the documentation
+  - Participate in [meetings](working-groups.md)
+  - Answer questions from other community members
+  - Submit feedback on issues and PRs
+  - Test, review, and verify releases and patches
+- Privileges:
+  - Invitations to contributor events
+  - Eligible to become an Organization Member
+
+### Organization Member
+
+Description: An Organization Member is an established contributor who regularly
+participates in the project. Organization Members have privileges in project
+repositories.
+
+An Organization Member must meet the responsibilities and has the requirements
+of a Contributor, plus:
+
+- Responsibilities include:
+  - Continues to contribute regularly, as demonstrated by having at least 15
+    contributions a year, as demonstrated by
+    [the Tekton devstats dashboard](https://tekton.devstats.cd.foundation/d/9/developer-activity-counts-by-repository-group-table?orgId=1&var-period_name=Last%20year&var-metric=contributions&var-repogroup_name=All&var-country_name=All).
+- Requirements:
+  - Must have successful contributions to the project, including at least one of
+    the following:
+    - Authored and Reviewed 5 PRs,
```

Comment:
nit are these separate requirements? e.g. do i need to author 5 PRs, and review 5 PRs? or a combo (e.g. author 2, review 3)?